seacatauth/external_login/handler.py

```diff
@@ -76,10 +76,18 @@ async def login(self, request):
 		sub = str(sub)
 
 		# Get credentials by sub
+		credentials_id = None
 		try:
 			el_credentials = await self.ExternalLoginService.get(login_provider_type, sub)
 			credentials_id = el_credentials["cid"]
 		except KeyError:
+			# Credentials do not exist in Seacat Auth
+			L.log(asab.LOG_NOTICE, "Unknown external login sub.", struct_data={
+				"provider_type": provider.Type, "sub": sub})
+			if self.ExternalLoginService.RegistrationWebhookUri:
+				credentials_id = await self.ExternalLoginService.register_credentials_via_webhook(
```

Comment:
webhook payload should also contain all the authorize response parameters, mainly `state`